### DISCUSSION TITLE
Is this normal behavior or should I file an issue?

### DISCUSSION THREAD

**Discussion by AllardKatan** [hassio-addons/addon-vscode#909] [General]
See here: https://community.home-assistant.io/t/what-is-causing-this-memory-leak/814871/8

**Comment by frenck** [comment]
Issue exists: https://github.com/hassio-addons/addon-vscode/issues/565

**Reply by AllardKatan** [reply]
OK thanks, I searched for 'memory leak' and it didn't show up

**Reply by AllardKatan** [reply]
@frenck I see there hasn't been any action on the issue. Can I help by providing more information/logs/etc? I'm neither proficient enough at coding to solve this, nor bothered enough by this to learn the skills, but if I can help others who are with a lesser investment of time I'd be happy to.

**Reply by frenck** [reply]
🤷 That happens, anyone can contribute information or a fix; that is how open source works.

No one has provided a fix as of yet.
